resources/carrier/en/30.txt

```diff
@@ -12,9 +12,115 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-30690|Wind
-30693|Wind
+
+# 30 - Greece
+
+# Source(s):
+# -----------------------
+# EETT: http://www.eett.gr/opencms/opencms/EETT_EN/Electronic_Communications/Telecoms/Numbering/search.html?cat=esa_non
+# -----------------------
```

Comment:
Mention source as below,
Source:http://www.eett.gr/opencms/opencms/EETT_EN/Electronic_Communications/Telecoms/Numbering/search.html?cat=esa_non

No need of '------'